octavia-cli/octavia_cli/api_http_headers.py

```diff
@@ -23,9 +23,11 @@ class ApiHttpHeader:
     value: str
 
     def __post_init__(self):
+        self.name = str(self.name)
+        self.value = str(self.value)
         try:
-            assert isinstance(self.name, str) and self.name
-            assert isinstance(self.value, str) and self.value
+            assert len(self.name) > 0
+            assert len(self.value) > 0
```

Comment:
My concern with this is that most python types can be casted to string without error.
e.g passing a `dict` as name or value will result in a non-empty casted string...